src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Duke {
     public static void main(String[] args) {
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        boolean isContinue = true;
+        printBanner();
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        while (isContinue) {
+            System.out.print("> ");
+            line = in.nextLine();
+            Command cmd = getCommand(line);
+
+            switch(cmd) {
```

Comment:
Use of switch-case here is good coding quality. If if-else were to be used it may appear too messy